### PR TITLE
use /ping as statuscake healthcheck url

### PR DESCRIPTION
**Pull request by vigneshmsft**
### Changes proposed in this pull request

configure the `/ping` as the health check endpoint in statuscake.

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
~- [ ] Product review~

### CODE REVIEW

**Review by TobyRet — APPROVED** [review]

File: terraform/terraform_prod.tfvars [hunk 1]
```diff
@@ -1,7 +1,7 @@
 alerts =  {
 ftt = {
     website_name = "prod-find-postgraduate-teacher-training"
-    website_url   = "https://www.find-postgraduate-teacher-training.service.gov.uk/healthcheck"
+    website_url   = "https://www.find-postgraduate-teacher-training.service.gov.uk/ping"
     test_type     = "HTTP"
     check_rate    = 300
```
Comment (saliceti) [inline terraform/terraform_prod.tfvars:6]: Please reduce interval to 1min

File: .github/workflows/terraform.yml [hunk 2]
```diff
@@ -28,6 +28,11 @@ jobs:
     - name: Checkout the repository to the GitHub Actions runner
       uses: actions/checkout@v2
 
+    - name: Use Terraform 0.12.29
```
Comment (saliceti) [inline .github/workflows/terraform.yml:31]: This is not required in this PR as statuscake is not currently delivered via github actions. We should create a separate card and PR for that.

**Comment by saliceti** [comment]
Could you please also add 1 more confirmation by default to avoid false positives?
We added it in teaching vacancies: https://github.com/DFE-Digital/teacher-vacancy-service/blob/master/terraform/app/modules/statuscake/statuscake.tf#L11

**Review by saliceti — APPROVED** [review]
OK

**Pull request merged**